x/dsmr/p2p.go

```diff
@@ -124,6 +160,11 @@ func (c ChunkSignatureRequestVerifier[T]) Verify(
 	}
 
 	if err := c.verifier.Verify(chunk); err != nil {
+		for _, e := range verifierAppErrors {
+			if errors.Is(err, e.Error) {
+				return e.AppError
+			}
+		}
```

Comment:
Ah I see that this is needed to support testing for an individual error. Apologies for this comment as I didn't realize this was what was required to support it. Could we revert back to remove this special case error handling and simply check for `ErrInvalidChunk` ?